solidity/contracts/bridge/Bridge.sol

```diff
@@ -1227,6 +1284,62 @@ contract Bridge is Ownable {
         bank.transferBalance(treasury, outputsInfo.totalTreasuryFee);
     }
 
+    /// @notice Submits a wallet fraud challenge.
+    /// @param walletPublicKey The public key of the wallet in the uncompressed
+    ///                        and unprefixed format (64 bytes).
+    /// @param walletPubKeyHash 20-byte public key hash (computed using Bitcoin
```

Comment:
I think it can be computed using `walletPublicKey`. Do we need it to be passed from the outside?

Reply:
Done in #3139da05e1fa4f1c5f8482a72f93fc35d87ac4b2.
I copied the `EcdsaLib` for now.